airbyte-integrations/connectors/source-mysql/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/MySqlStrictEncryptSslCaCertificateSourceAcceptanceTest.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.io.airbyte.integration_tests.sources;
+
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.cdk.db.MySqlUtils;
+import io.airbyte.cdk.db.jdbc.JdbcUtils;
+import io.airbyte.cdk.integrations.base.ssh.SshHelpers;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.resources.MoreResources;
+import io.airbyte.protocol.models.v0.ConnectorSpecification;
+import java.io.IOException;
+
+public class MySqlStrictEncryptSslCaCertificateSourceAcceptanceTest extends AbstractMySqlStrictEncryptSslCertificateSourceAcceptanceTest {
+
+  @Override
+  protected ConnectorSpecification getSpec() throws Exception {
+    return SshHelpers.injectSshIntoSpec(Jsons.deserialize(MoreResources.readResource("expected_cloud_spec.json"), ConnectorSpecification.class));
+  }
```

Comment:
nit: move this method definition to the superclass, remove it from the other subclass